sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuRowToColumnarExec.scala

```diff
@@ -806,6 +817,8 @@ case class GpuRowToColumnarExec(child: SparkPlan, goal: CoalesceSizeGoal)
   }
 
   override lazy val additionalMetrics: Map[String, GpuMetric] = Map(
+    "semaphoreWait" -> createNanoTimingMetric(DEBUG_LEVEL, "Semaphore wait time"),
```

Comment:
Curious why this isn't defined in `GpuMetric` like the others?  I would expect this to start showing up in other execs that acquire the semaphore.